viv-wallet-app/.env

```diff
@@ -2,5 +2,4 @@ NODE_ENV=production
 VUE_APP_BACKEND_BASE_URL=/api/v1
 VUE_APP_REQUEST_TIMEOUT_MS=10000
 VUE_APP_APPS_URL=https://apps.uat.xcomponent.com
-VUE_APP_DEV_JWT=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ2aXYtd2FsbGV0Ijoie1wicm9sZXNcIjpbXCJDT01QQU5ZX0FETUlOSVNUUkFUT1JcIixcIkVYUEVSVElTRV9NQU5BR0VSXCJdfSIsInR5cGUiOiJSZWd1bGFyVXNlciIsImlzcyI6Ilg0QiIsInVzZXIiOiJUaMOpb3BoaWxlIE1PTlRHT01FUlkiLCJzcmMiOiJkYXRhYmFzZSJ9.iy8owTAaCQ0WWeICGw8AZY_4pb8w3ptwdUzlqCWEXs8
```

Comment:
it looks that a documentation of how we are supposed to set this variable and where the token should come from should be added somewhere. a README?